packages/itwin/measure-tools/src/measurements/DistanceMeasurement.ts

```diff
@@ -545,35 +574,41 @@ export class DistanceMeasurement extends Measurement {
         name: "DistanceMeasurement_Slope",
         value: fSlope,
       },
-      {
-        label: MeasureTools.localization.getLocalizedString(
-          "MeasureTools:tools.MeasureDistance.delta_x"
-        ),
-        name: "DistanceMeasurement_Dx",
-        value: fDeltaX,
-      },
-      {
-        label: MeasureTools.localization.getLocalizedString(
-          "MeasureTools:tools.MeasureDistance.delta_y"
-        ),
-        name: "DistanceMeasurement_Dy",
-        value: fDeltaY,
-      },
-      {
-        label: MeasureTools.localization.getLocalizedString(
-          "MeasureTools:tools.MeasureDistance.startCoordinates"
-        ),
-        name: "DistanceMeasurement_StartPoint",
-        value: fStartCoords,
-      },
-      {
-        label: MeasureTools.localization.getLocalizedString(
-          "MeasureTools:tools.MeasureDistance.endCoordinates"
-        ),
-        name: "DistanceMeasurement_EndPoint",
-        value: fEndCoords,
-      }
     );
+
+    if (this._sheetToWorldScale === undefined) {
```

Comment:
Is this needed? Alternatively, we can strip out some of those fields, or put values for both
like an entry for
`Delta Y`
and
`Delta Y (sheet)`

Might want to double check with product management what they expect.